deploy/operator/setup_hive.sh

```
@@ -3,6 +3,7 @@
 __dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 source ${__dir}/common.sh
 source ${__dir}/utils.sh
+source ${__dir}/mirror_utils.sh
```

Shellcheck (0.11.0)

[info] Not following: ./mirror_utils.sh was not specified as input (see shellcheck -x).

(SC1091)